### PR TITLE
[luci] Extract code to quantize special activation

### PR DESCRIPTION
**Pull request by jinevening**
This extracts code to quantize special activation.

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

---
Related to: https://github.com/Samsung/ONE/issues/8512
Draft PR: https://github.com/Samsung/ONE/pull/8614

### CODE REVIEW

**Comment by jinevening** [comment]
I extract this code to use it in another file.

File: compiler/luci/pass/src/QuantizationUtils.cpp [hunk 1]
```diff
@@ -315,4 +315,85 @@ uint32_t cal_offset(loco::TensorShape &dimension, uint32_t *indices)
          indices[2] * dimension.dim(3).value() + indices[3];
 }
 
+ActivationQType activation_qtype(CircleNode *node)
```
Comment (seanshpark) [inline compiler/luci/pass/src/QuantizationUtils.cpp:318]: (optional) `activation_qtype(const CircleNode *node)` ?

**Review by seanshpark — APPROVED** [review]
LGTM thank you!

File: compiler/luci/pass/src/QuantizationUtils.cpp [hunk 2]
```diff
@@ -315,4 +315,85 @@ uint32_t cal_offset(loco::TensorShape &dimension, uint32_t *indices)
          indices[2] * dimension.dim(3).value() + indices[3];
 }
 
+ActivationQType activation_qtype(const CircleNode *node)
+{
+  auto fused_act_node = dynamic_cast<const CircleNodeMixin<CircleNodeTrait::FusedActFunc> *>(node);
+  if (fused_act_node && fused_act_node->fusedActivationFunction() == FusedActFunc::TANH)
+    return ActivationQType::PreDefinedValue;
```
Comment (mhs4670go) [inline compiler/luci/pass/src/QuantizationUtils.cpp:322]: This is not covered at below `case` statement?

Comment (jinevening) [reply compiler/luci/pass/src/QuantizationUtils.cpp:322]: This code checks fused activation function. For example, Conv + Tanh or DConv + Tanh.

The below code checks node itself is Tanh.

**Review by mhs4670go — APPROVED** [review]
LGTM

**Pull request merged**